### PR TITLE
[REFACTOR] Force require at top level

### PR DESCRIPTION
**Pull request by KamiKillertO**

### CODE REVIEW

File: lib/default_conf.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-let {version} = require('../index')
+const { version } = require('../package.json')
+
```
Comment (ryuran) [inline lib/default_conf.js:2]: Useless new empty line (should be linted too)

Comment (KamiKillertO) [reply lib/default_conf.js:2]: The eslint-peopledoc-config has no rule for this 🤔 . Personally I like having a empty line between import/require and the rest of the code for readability. Maybe we can ask the tribe waht they prefer

Comment (ryuran) [reply lib/default_conf.js:2]: I agree for one empty line, not 2 ^^

Comment (KamiKillertO) [reply lib/default_conf.js:2]: Oh yes there's two newline 😅

**Pull request merged**